src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { Modal } from 'react-bootstrap';
+import browser from 'detect-browser';
+
+
+interface IBrowserState {
+    show: boolean;
+    name: string;
+    version: string;
+}
+
+export default class UnsupportedBrowserModal extends React.Component<{}, IBrowserState> {
+
+  constructor() {
+      super();
+
+      if (!browser)  {
+          this.state = {
+              name: "Unsupported",
+              show: true,
+              version: ""
+          };
+      } else {
+          this.state = {
+              name: browser.name,
+              show: false,
+               version: browser.version
+          };
+      }
+  }
+
+  componentDidMount() {
+      this.handleUnsupportedBrowsers(this.state.name, this.state.version);
+  }
+
+  handleUnsupportedBrowsers(name:string, version: string) {
+      name = name.toLowerCase();
+      const localStorage = window.localStorage.browserError || false;
+      if (localStorage === true) {
+          this.setState({show: false});
+      } else if (String(name) === 'unsupported') {
+          window.localStorage.browserError = true;
+          this.setState({show: true});
+      } else if (!(name === "chrome" || name === "firefox" || name === "edge") || (String(name) === "ie" && Number(version.slice(0,2)) < 11)) {
+          window.localStorage.browserError = true;
+          this.setState({show: true});
+      } else {
+          this.setState({show: false});
```

Comment:
Because I thought you said we only wanted to do this check on the first occurrence, meaning that the modal should not sure, regardless of the browser or its version, after the site first loads.